sites/partners/cypress/integration/05-paperApplication.spec.ts

```diff
@@ -9,7 +9,7 @@ describe("Paper Application Tests", () => {
 
   beforeEach(() => {
     cy.visit("/")
-    cy.getByTestId("listing-status-cell").eq(1).click()
+    cy.get(`[row-id="0"]`).find(`[col-id="status"]`).contains("Open").click()
```

Comment:
If possible, we want to get items by test ids to rely less on implementation details. If we changed the row-id here in the component, it would likely break the test, and potentially cause a false negative. It's clearer in the components to use a test id so we know where to put it if we do a refactor.

Reply:
Make sense! I changed this while testing broken cypress tests, agree with you, restoring the previous declaration, thanks!